### DISCUSSION TITLE
How to authenticate using LDAP/AD in MISP?

### DISCUSSION THREAD

**Discussion by guaar** [MISP/MISP#8926] [Q&A]
Is there an official guidance or article on how to integrate LDAP/AD authentication with MISP?

**Comment by ranjbassi** [comment]
bump... Anything? I am in a similar position and would like to know more, having a quick look, documentation is light.

**Comment by ranjbassi** [comment]
The best I have found are:

Appendix F -https://circl.lu/doc/misp/appendices/#appendix-f-ldap-authentication - Red Hat 
https://hdoc.csirt-tooling.org/ALBUDpXOQe-4XX4n__yUIg?view - Ubuntu
